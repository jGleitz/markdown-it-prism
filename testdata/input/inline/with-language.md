`System.out.println("bar")`{language=java}
