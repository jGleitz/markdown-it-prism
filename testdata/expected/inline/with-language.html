<p><code class="language-java"><span class="token class-name">System</span><span class="token punctuation">.</span>out<span class="token punctuation">.</span><span class="token function">println</span><span class="token punctuation">(</span><span class="token string">"bar"</span><span class="token punctuation">)</span></code></p>
